src/schemas/json/buf.gen.json

```diff
@@ -0,0 +1,316 @@
+{
+  "$comment": "https://buf.build/docs/configuration/v1/buf-gen-yaml",
+  "$id": "https://json.schemastore.org/buf.gen.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "buf.gen.yaml",
+  "description": "buf.gen.yaml is a configuration file used by the buf generate command to generate integration code for the languages of your choice.",
+  "type": "object",
+  "properties": {
+    "version": {
+      "$comment": "https://buf.build/docs/configuration/v1/buf-gen-yaml#version",
+      "description": "Required. Defines the current configuration version. Accepted values are v1beta1 and v1.",
+      "type": "string",
+      "enum": ["v1beta1", "v1"]
+    },
+    "plugins": {
+      "$comment": "https://buf.build/docs/configuration/v1/buf-gen-yaml#plugins",
+      "description": "Required. Each entry in the plugins key is a protoc plugin configuration. Plugins are invoked in parallel and their outputs are written in the order you specify here.",
```

Comment:
Most of these descriptions are lifted directly from these docs, which I'm assuming is kosher.

I'm not sure if it's best practice to leave these as plaintext or attempt to "markdownify" some of the text - there's definitely bold, italics and links in the source text. I basically left these as plaintext, except adding a "See <link>." in the case that there were more details in the docs.

Reply:
Yeah, it is recommended to make the description plaintext-only. You can use [markdownDescription](https://code.visualstudio.com/docs/languages/json#_use-rich-formatting-in-hovers) if you wish to use markdown. Although, I'm not sure how portable that is

Reply:
I'll leave these as plaintext, thanks.